09week/hackernews/script.js

```diff
@@ -1 +1,90 @@
 'use strict';
+
+class Hacker extends React.Component {
+  constructor(props){
+    super(props);
+
+    this.state = {
+      list: []
+    }
+  }
+
+  componentDidMount() {
+    const articleList = [];
+    for(let i=1; i<151; i++){
+      const url = `https://hacker-news.firebaseio.com/v0/item/${i}.json?print=pretty`;
```

Comment:
this is fetching the first 150 articles of the site, and not the current articles.
To do that you would have had to use ```https://hacker-news.firebaseio.com/v0/topstories.json?print=pretty``` to get the top stories first and use the ids that are returned to fetch the story details